app/controllers/users_controller.rb

```diff
@@ -19,30 +19,31 @@ def user_guardians
     if admin?
       @guardians = current_user.guardians.all
       raise ActiveRecord::RecordNotFound unless @guardians
+
       render json: @guardians, status: :ok
     else
       render json: { error: 'Not Allowed' }, status: :unauthorized
     end
   end
 
   def add_guardian
-    if admin?
-      if User.find_by_email(user_params[:email])
-        render json: { error: 'Email Exist , try a diffrent one' }, status: :not_acceptable
+    return unless admin?
```

Comment:
Why don't you just create a guardians_cotroller?

Reply:
@nidalaa. Done